python/cudf/cudf/tests/test_list.py

```diff
@@ -926,3 +927,29 @@ def test_list_iterate_error():
 def test_list_struct_list_memory_usage():
     df = cudf.DataFrame({"a": [[{"b": [1]}]]})
     assert df.memory_usage().sum() == 16
+
+
+def test_empty_nested_list_uninitialized_offsets_memory_usage():
```

Comment:
Would it be possible to re-write this test in-terms of a public API rather than accessing column constructors?

Reply:
I tried finding a public API (constructor) that would create an empty offset column, but I think the cudf Python layer is good at consistently ensuring the offset column has at least one entry.

There are operations that go through libcudf that will return an empty offset column (e.g. `.iloc[0:0]`), but I wanted to ensure we test something that has the least chance to change if a public API implementation changes.

Reply:
Got it.